recipes/tcb-span/all/conanfile.py

```diff
@@ -18,6 +17,9 @@ class TcbSpanConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def configure(self):
+        tools.check_min_cppstd(self, "11")
```

Comment:
We make this mistake often https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough

```suggestion
        if self.settings.compiler.get_safe("cppstd"):
            tools.check_min_cppstd(self, 11)
```